build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir -p content
+pip install -r requirements.txt
+python ./openshift_versions/cmd/versions.py
+cp index.html content/
```

Comment:
Ideally the index.html only needs to be updated if it is changed, like:
```
python ./openshift_versions/cmd/versions.py && \
  cp index.html content/
```

Reply:
Let me amend it

Reply:
As we run in a container with no prior data, we require to get the full content generated or it will be overwriten with empty page.

We're discussing alternatives as trying to use caching of generated files and only copy over when data changes

Reply:
Check latest commit using caches for storing index.html and versions.json